validator.rb

```diff
@@ -0,0 +1,19 @@
+class Validator
+
+  def is_a_word? word 
```

Comment:
This if/else is unnecessary, `word.match(/^[a-zA-Z]+$/) && word.length > 1` by itself will either return as true or false.